### ISSUE TITLE
Fix and reenable FixesAreReportedForCompilerErrors()

### ISSUE THREAD

**Issue by KirillOsenkov** [mono/monodevelop#6981]
https://github.com/mono/monodevelop/blob/9d0c45dfdc1a8853d05ad4f4efffdf3fe90dc7be/main/tests/MonoDevelop.CSharpBinding.Tests/MonoDevelop.CSharpBinding.Refactoring/CSharpCodeActionEditorExtensionTests.cs#L78

```
1) FixesAreReportedForCompilerErrors (MonoDevelop.CSharpBinding.Refactoring.CSharpCodeActionEditorExtensionTests.FixesAreReportedForCompilerErrors)
     Expected string length 19 but was 14. Strings differ at index 0.
  Expected: "Implement interface"
  But was:  "Fix formatting"
  -----------^

  at MonoDevelop.Refactoring.Tests.CodeActionEditorExtensionTestBase.AssertExpectedCodeFixes (MonoDevelop.Refactoring.Tests.ExpectedCodeFixes expected, MonoDevelop.Ide.Gui.Document doc) [0x000cb] in /Users/vsts/agent/2.144.2/work/1/s/monodevelop/main/tests/MonoDevelop.Refactoring.Tests/CodeActionEditorExtensionTestBase.cs:63 
  at MonoDevelop.CSharpBinding.Refactoring.CSharpCodeActionEditorExtensionTests+<>c__DisplayClass4_0.<FixesAreReportedForCompilerErrors>b__0 (System.Int32 remainingUpdates, MonoDevelop.Ide.Gui.Document doc) [0x00065] in /Users/vsts/agent/2.144.2/work/1/s/monodevelop/main/tests/MonoDevelop.CSharpBinding.Tests/MonoDevelop.CSharpBinding.Refactoring/CSharpCodeActionEditorExtensionTests.cs:103 
  at MonoDevelop.Refactoring.Tests.ResultsEditorExtensionTestBase+<>c__DisplayClass7_0.<OnRunTest>b__0 (MonoDevelop.Ide.Gui.Document doc, System.Threading.Tasks.TaskCompletionSource`1[TResult] tcs) [0x00039] in /Users/vsts/agent/2.144.2/work/1/s/monodevelop/main/tests/MonoDevelop.Refactoring.Tests/ResultsEditorExtensionTestBase.cs:145 
  at MonoDevelop.Refactoring.Tests.ResultsEditorExtensionTestBase+<>c__DisplayClass2_0`1[T].<GatherDiagnosticsNoDispose>b__0 (System.Object o, System.EventArgs args) [0x00031] in /Users/vsts/agent/2.144.2/work/1/s/monodevelop/main/tests/MonoDevelop.Refactoring.Tests/ResultsEditorExtensionTestBase.cs:77 
  at MonoDevelop.Refactoring.Tests.ResultsEditorExtensionTestBase.GatherDiagnosticsNoDispose[T] (System.String input, System.Func`3[T1,T2,TResult] callback) [0x001d2] in /Users/vsts/agent/2.144.2/work/1/s/monodevelop/main/tests/MonoDevelop.Refactoring.Tests/ResultsEditorExtensionTestBase.cs:90 
  at MonoDevelop.Refactoring.Tests.ResultsEditorExtensionTestBase.GatherDiagnostics[T] (System.String input, System.Func`3[T1,T2,TResult] callback) [0x00035] in /Users/vsts/agent/2.144.2/work/1/s/monodevelop/main/tests/MonoDevelop.Refactoring.Tests/ResultsEditorExtensionTestBase.cs:100 
  at MonoDevelop.Refactoring.Tests.ResultsEditorExtensionTestBase.RunTest (System.Int32 expectedUpdates, System.String input, System.Func`3[T1,T2,TResult] onUpdate) [0x00053] in /Users/vsts/agent/2.144.2/work/1/s/monodevelop/main/tests/MonoDevelop.Refactoring.Tests/ResultsEditorExtensionTestBase.cs:136 
  at MonoDevelop.CSharpBinding.Refactoring.CSharpCodeActionEditorExtensionTests.FixesAreReportedForCompilerErrors () [0x00117] in /Users/vsts/agent/2.144.2/work/1/s/monodevelop/main/tests/MonoDevelop.CSharpBinding.Tests/MonoDevelop.CSharpBinding.Refactoring/CSharpCodeActionEditorExtensionTests.cs:98 ```

> VS bug [#778962](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/778962)

**Comment by Therzok** [comment]
Probably needs 1 more update

**Comment by sandyarmstrong** [comment]
I have a fix for this by applying ordering to the result lists (expected and actual).

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #119 merged by: Sandy Armstrong
Author: xamarin
Commit: 765c9ab63999348b74bb1bbdcb4a4b662064b750 (xamarin/monodevelop)
